internal/venice-common/src/main/resources/avro/PartitionState/v14/PartitionState.avsc

```diff
@@ -186,6 +186,12 @@
         "avro.java.string": "String"
       },
       "default": {}
+    },
+    {
+      "name": "transformerClassHash",
+      "doc": "An integer hash code of the DaVinci record transformer for compatibility checks during BLOB transfer",
+      "type": ["null", "int"],
+      "default": null
```

Comment:
Nit: DaVinci record transformer -> DaVinciRecordTransformer

This will also be utilized when blob transfer isn't enabled. Can we change the phrasing to "during bootstrapping?"